From ae851f3006be19112fdf5ccdf17f11b90d2eab30 Mon Sep 17 00:00:00 2001
From: Nicholas Devenish <ndevenish@gmail.com>
Date: Wed, 18 Sep 2024 22:07:11 +0100
Subject: [PATCH 5/5] Fix dependency names for conda-forge

---
 pyproject.toml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 007963a..1b54322 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -35,14 +35,14 @@ classifiers = [
 ]
 
 dependencies = [
-    "cadquery-ocp >= 7.7.0",
+    "cadquery >= 2.2.0",
     "typing_extensions >= 4.6.0, <5",
     "numpy >= 1.24.1, <2",
     "svgpathtools >= 1.5.1, <2",
     "anytree >= 2.8.0, <3",
     "ezdxf >= 1.1.0, < 2",
     "ipython >= 8.0.0, <9",
-    "py-lib3mf >= 2.3.1",
+    "lib3mf-python >= 2.3.1",
     "ocpsvg",
     "trianglesolver"
 ]
-- 
2.46.0

